Casks/dvc.rb

```diff
@@ -0,0 +1,24 @@
+cask "dvc" do
+  version "2.33.2"
+  sha256 "ceebcb6be244f6fdb5c6329ccd0ceb2cc25c727c6a4929f7d690716a708b90af"
+
+  # github.com/iterative/dvc/ was verified as official when first introduced to the cask
+  url "https://github.com/iterative/dvc/releases/download/#{version}/dvc-#{version}.pkg"
```

Comment:
github releases are not official packages and are unsigned.
```suggestion
  url "https://dvc.org/download/osx/dvc-#{version}.pkg"
```